module.conf

```diff
@@ -0,0 +1,233 @@
+#
+# search-vss-max-resize 0
+
+# Number of worker threads to use for query processing and background tasks.
+# numeric, valid range: [0, 16], default: 0
+# This configuration also affects the number of connections per shard.
+#
+# search-workers 0
+
+# The number of high priority tasks to be executed at any given time by the
+# worker thread pool, before executing low priority tasks. After this number
+# of high priority tasks are being executed, the worker thread pool will
+# execute high and low priority tasks alternately.
+# numeric, valid range: [0, LLONG_MAX], default: 1
+#
+# search-workers-priority-bias-threshold 1
+
+# Load extension scoring/expansion module. Immutable.
+# string, default: ""
+#
+# search-ext-load ""
+
+# Path to Chinese dictionary configuration file (for Chinese tokenization). Immutable.
+# string, default: ""
+#
+# search-friso-ini ""
+
+# Action to perform when search timeout is exceeded (choose RETURN or FAIL).
+# enum, valid values: ["return", "fail"], default: "return"
+#
+# search-on-timeout return
+
+# Determine whether some index resources are free on a second thread.
+# bool, default: yes
+#
+# search-_free-resource-on-thread yes
+
+# Enable legacy compression of double to float.
+# bool, default: no
+#
+# search-_numeric-compress no
+
+# Disable print of time for ft.profile. For testing only.
+# bool, default: yes
+#
+# search-_print-profile-clock yes
+
+# Intersection iterator orders the children iterators by their relative estimated
+# number of results in ascending order, so that if we see first iterators with
+# a lower count of results we will skip a larger number of results, which
+# translates into faster iteration. If this flag is set, we use this
+# optimization in a way where union iterators are being factorize by the number
+# of their own children, so that we sort by the number of children times the
+# overall estimated number of results instead.
+# bool, default: no
+#
+# search-_prioritize-intersect-union-children no
+
+# Set to run without memory pools.
+# bool, default: no
+#
+# search-no-mem-pools no
+
+# Disable garbage collection (for this process).
+# bool, default: no
+#
+# search-no-gc no
+
+# Enable commands filter which optimize indexing on partial hash updates.
+# bool, default: no
+#
+# search-partial-indexed-docs no
+
+# Disable compression for DocID inverted index. Boost CPU performance.
+# bool, default: no
+#
+# search-raw-docid-encoding no
+
+# Number of search threads in the coordinator thread pool.
+# numeric, valid range: [1, LLONG_MAX], default: 20
+#
+# search-threads 20
+
+# Timeout for topology validation (in milliseconds). After this timeout,
+# any pending requests will be processed, even if the topology is not fully connected.
+# numeric, valid range: [0, LLONG_MAX], default: 30000
+#
+# search-topology-validation-timeout 30000
+
+
+# Enable unstable new features for early adopters. Use at your own risk.
+#
+# bool, default: no
+# search-enable-unstable-features false
```

Comment:
Why not keep the `search-enable-unstable-features` out of the conf file? Not relevant for most users,
and if needed they can add `search-enable-unstable-features yes` or set it using `CONFIG SET`

Reply:
I don't have a strong opinion here; many of the configurations here are niche. I thought we would include all of them (except for ones that are pure debug configs)

Reply:
I am inclining towards making this specific config less noticeable (excluded from conf file)
If you prefer to keep it, change the default from `false` to `no`